src/compas/datastructures/mesh/explode.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from compas.topology import connected_components
+
+__all__ = [
+    'mesh_disconnected_vertices',
+    'mesh_disconnected_faces',
+    'mesh_explode',
+]
+
+def mesh_disconnected_vertices(mesh):
+    """Get the disconnected vertex groups in a mesh.
+
+    Parameters
+    ----------
+    mesh : Mesh
+        A mesh.
+
+    Returns
+    -------
+    parts : list
+        The list of disconnected vertex groups.
+
+    """
+
+    return connected_components(mesh.adjacency)
+
+def mesh_disconnected_faces(mesh):
```

Comment:
can you correct the spacing.
two empty lines before and after function and class definitions...